system/lib/wasmfs/syscalls.cpp

```diff
@@ -1127,7 +1127,8 @@ int __syscall_fchmodat(int dirfd, intptr_t path, int mode, ...) {
     return -EINVAL;
   }
   // TODO: Handle AT_SYMLINK_NOFOLLOW once we traverse symlinks correctly.
```

Comment:
This TODO line can be removed now, I think, as passing in flags as this PR does to the modified call below will handle that flow.